apps/ui/src/components/chip/chip.tsx

```diff
@@ -0,0 +1,88 @@
+import { useRef, useLayoutEffect, useMemo } from 'react';
+import _ from 'lodash';
+
+import type { ChipsProperties } from "./types";
+import { toDefaults, toClasses, toRadius } from "./helpers";
+import { Icon } from '../icon/index';
+import "./styles/index.css";
+
+/**
+ * chip component
+ *
+ * @param {Properties} properties - refers to chip properties
+ *
+ * @example
+ * <Chip role="assist" color="primary" variant="outlined"  onClick={handler}>Content</Chip>
+ *
+ * @returns {React.JSX.Element} element
+ */
+
+export default function Chip(properties?: ChipsProperties) {
+	const defaults = toDefaults(properties);
+	const reference = useRef<HTMLElement | null>(null);
+	const Element = defaults.href ? 'a' : 'span';
```

Comment:
I think `href ` is used to url, could you elaborate a summary about this solution ?